contracts/staking/RewardDelegators.sol

```diff
@@ -135,6 +139,14 @@ contract RewardDelegators is
     event ClusterRegistryUpdated(address _updatedClusterRegistry);
     event PONDAddressUpdated(address _updatedPOND);
 
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    bytes32 public immutable pondToken;
+
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    bytes32 public immutable mpondToken;
+
+    uint256 public constant thresholdForSelection = 500_000; // 0.5 million POND or equivalent
```

Comment:
This might be a variable, so let's not set it to constant

Reply:
add a function to update it

Reply:
It probably makes more sense to move this variable to EpochSelector contract as it is related closely to that than delegations as such. Although I do understand that it might cost slightly higher for cases when cluster delegation is < threshold